src/toil/server/wes/toil_backend.py

```diff
@@ -273,8 +273,8 @@ def get_run_log(self, run_id: str) -> Dict[str, Any]:
         stdout = ""
         stderr = ""
         if os.path.isfile(os.path.join(run.work_dir, 'stdout')):
-            stdout = f"{self.base_url}/toil/wes/v1/logs/{run_id}/stdout"
-            stderr = f"{self.base_url}/toil/wes/v1/logs/{run_id}/stderr"
+            stdout = f"{flask_request.host_url}toil/wes/v1/logs/{run_id}/stdout"
+            stderr = f"{flask_request.host_url}toil/wes/v1/logs/{run_id}/stderr"
```

Comment:
This is a good idea.

Reply:
I noticed that you did something similar in [#4039](https://github.com/DataBiosphere/toil/pull/4039/files#diff-e05a9b10abd96d5732ff2a3577a21843f0c6efa1074f2e44d752670449124edbR290-R297), which will cause some conflicts.